### ISSUE TITLE
Support for "child themes"

### ISSUE THREAD

**Issue by kavonjon** [MukurtuCMS/mukurtucms#106]
It would be really helpful for communities that don't have continual access to people with drupal knowledge to be able to implement customizations in their own instance of this CMS without needed to re-impliment them every time the core MukurtuCMS is updated. I'm thinking of something like wordpress child themes. You can edit the theme files in a child theme and those modifications persist when the main theme is updated.

**Comment by steve-taylor-wsu** [comment]
I agree that would be ideal. It is currently possible to make child themes from the existing "Mukurtu Starter" theme (https://www.drupal.org/docs/7/theming/creating-a-sub-theme), similar to WordPress. We are working on an update to the stock Mukurtu theme to make it easier to extend, and we are changing how we package things to make site updates less disruptive to local changes. With the theme update, we are hoping to provide more Mukurtu specific documentation and examples of how creating a sub-theme works. Thanks.
